spec/support/dom_matcher.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+include Rails::Dom::Testing::Assertions::DomAssertions
+
+RSpec::Matchers.define :be_same_dom do |expected|
```

Comment:
This is nice, wish I had this for hashtags! Just pushing a fix because it says `failure_message_for_should` is deprecated and to use `failure_message` instead

Reply:
Thank you!